Elizabeth_Theme/src/css/_custom.scss

```diff
@@ -0,0 +1,30 @@
+@import 'colors';
+@import 'components/buttons';
+
+@import 'layout/navigation';
+@import 'layout/welcome-page';
+@import 'layout/footer';
+
+@import 'components/map';
+
+body {
+  &.has-control-menu {
+
+    header {
+      margin-top: 0; 
+
+      @include media-breakpoint-down(sm) {
+        margin-top: 48px;
+      }  
+    }
+  }
+  
+  //RESPONSIVE BACKGROUND
+  background-attachment: fixed; //Fixes the image in the viewport so that it doesn't move when the content height is greater than the image height
```

Comment:
Space required after `//` for inline comments.

Reply:
We should not use contractions for comments.  We should be as clear as possible.   Are comments even needed for this?